docs/team/oliviajohansen.md

```diff
@@ -52,7 +52,9 @@ Given below are my contributions to the project.
 
 * **Enhancements to existing features**:
   * Added support for quantity for ingredients (Pull requests [\#82](https://github.com/AY2021S1-CS2103T-W10-2/tp/pull/82))
-    * Justification: This feature allow users to better keep track of quantity of ingredients in the fridge.
+    * Justification: This feature allow users to better keep track of quantity of ingredients in the fridge. The
+     flexible format of quantity (fraction, decimal or integers followed by units or just a string) allowed
```

Comment:
allows users?

Reply:
whats wrong?

Reply:
ohoh the 2nd allow, the flexible format... just a string) allowed -> allows to match 1st allow?

Reply:
oo yes